autoscheduler/frontend/src/components/LandingPage/SelectTerm.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
```

Comment:
GitHub Actions is showing some linting errors in this file that need to be fixed before merging